browser-test/src/admin/admin_question_list.test.ts

```diff
@@ -31,25 +31,25 @@ test.describe('Admin question list', () => {
 
     // Most recently added question is on top.
     await expectQuestionListElements(adminQuestions, [
-      questionTwoPublishedText,
-      questionOnePublishedText,
+      'question list test question two\n',
```

Comment:
suggestion (blocking): Reference the const here and then add `+ \n`. That way if someone changes the text for the const, they don't have to update it in multiple places.

Reply:
I've gotten into the habit of not using variables in the expect clause and I'm interested in discussing whether that makes sense for these unit tests! 

My reasoning is that by using the literal and not obfuscating them with a variable name is it directly conveys the expected value instead of having to scroll up and find what the variable matches. I also get "variable blindness" in that the names "questionOnePublishedText" and "questionTwoPublishedText" are so similar that it takes me a second or two to understand which text is expected. I think the literal strings themselves are easier to read because of the spaces :) 

But I think variables make sense if we want to make multiple assertions about a single interaction / object / class or property or complex interactions. I also believe consistency is important! So don't want to change it only here and have it be different from the current testing pattern. 

Let me know what you think!

Reply:
That makes sense, and I actually hadn't thought about that before! I don't have a strong preference as long as we're consistent. I'd say go with either for now and let's chat about it at the next eng meet. It looks like there are some places where you're still referencing the var name in the expect statement though (eg `questionOneDraftText + '\n',`), so please update those to strings if you go that route.